.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: Build
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+jobs:
+    build:
+        name: PHP ${{ matrix.php-versions }} ${{ matrix.composer-args }}
+        strategy:
+            fail-fast: false
+            matrix:
+                operating-system: [ubuntu-latest]
+                php-versions: ['7.2', '7.3', '7.4']
+                composer-args: ['', '--prefer-lowest']
+                include:
+                    - operating-system: ubuntu-latest
+                      php-versions: '8.0'
+                      composer-args: '--ignore-platform-reqs --no-scripts'
+                    - operating-system: ubuntu-latest
+                      php-versions: '8.0'
+                      composer-args: '--prefer-lowest --ignore-platform-reqs --no-scripts'
```

Comment:
Just making sure, are these composer flags still necessary here?

Reply:
`--ignore-platform-reqs` might not be, I can test though